ShiftsLogger.Kronyer/ShiftLoggerConsole/DataAccess.cs

```diff
@@ -0,0 +1,294 @@
+﻿using Newtonsoft.Json;
+using RestSharp;
+using Spectre.Console;
+
+namespace ShiftLoggerConsole;
+
+public class DataAccess
+{
+    public static void RegisterNewWorker()
+    {
+        try
+        {
+            string name = AnsiConsole.Ask<string>("Type the name").Trim().ToLower();
+
+            while (name == null)
+            {
+                Console.WriteLine("Name can't be empty...");
+                name = AnsiConsole.Ask<string>("Type the name:").Trim().ToLower();
+            }
+
+            var client = new RestClient("http://localhost:5259/");
+            var request = new RestRequest("create", Method.Post);
+            string jsonBody = "\"" + name + "\"";
+            request.AddParameter("application/json", jsonBody, ParameterType.RequestBody);
+            var response = client.Execute(request);
+
+            if (response.IsSuccessful)
+            {
+                Console.WriteLine("Worker created!");
+            }
+            else
+            {
+                Console.WriteLine($"ERROR: {response.ErrorMessage}");
+            }
+
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine(ex.Message);
+        }
+        finally
+        {
+            UserInterface.RunMenu();
+        }
+    }
+
+    public static void StartShift()
+    {
+        int workers = CountWorkers();
+        if (workers == 0)
+        {
+            Console.WriteLine("There is no worker in Db, please create one first");
+            Console.ReadKey();
+            UserInterface.RunMenu();
+        }
+        AnsiConsole.MarkupLine("Start work:");
+
+        string name = AnsiConsole.Ask<string>("Type the name:").Trim().ToLower();
+
+        while (name == null)
+        {
+            Console.WriteLine("Name can't be empty...");
+            name = AnsiConsole.Ask<string>("Type the name:").Trim().ToLower();
+        }
+        if (!WorkerExists(name))
+        {
+            Console.WriteLine("This worker is not registered yet, please register...");
+            Console.ReadKey();
+            UserInterface.RunMenu();
+        }
+
+        try
+        {
+            var client = new RestClient("http://localhost:5259/");
```

Comment:
I would suggest reading the base url from appsettings. In case the address changes, you only need to edit it once.